build/android/src/main/res/layout/assetcopy.xml

```diff
@@ -18,7 +18,7 @@
         android:ellipsize="middle" 
         android:singleLine="true"
         android:layout_gravity="center_horizontal"
-        android:text="preparing media ..."
+        android:text="@string/preparing_media"
```

Comment:
There is no good reason to do it the "complicated" way either. Why not just disable the lint.

But idk, its no big issue.